source/shared/cpp/ObjectModel/SharedAdaptiveCard.cpp

```diff
@@ -185,9 +185,11 @@ std::shared_ptr<ParseResult> AdaptiveCard::Deserialize(const Json::Value& json,
     std::string fallbackText = ParseUtil::GetString(json, AdaptiveCardSchemaKey::FallbackText);
     std::string language = ParseUtil::GetString(json, AdaptiveCardSchemaKey::Language);
     std::string speak = ParseUtil::GetString(json, AdaptiveCardSchemaKey::Speak);
-
+    
+#if !defined(__APPLE__)
```

Comment:
nit: I suggest changing it to #ifndef for consistency with the rest of the codes.